apps/Gruntfile.js

```diff
@@ -63,6 +64,7 @@ module.exports = function (grunt) {
 
   var ace_suffix = envOptions.dev ? '' : '-min';
   var dotMinIfNotDev = envOptions.dev ? '' : '.min';
+  var piskelRoot = String(child_process.execSync('`npm bin`/piskel-root')).replace(/\s+$/g,'');
```

Comment:
The newline in the output of this utility caused issues within `grunt-contrib-copy`; I'll simplify this as soon as I fix up the utility.